DNN Platform/DotNetNuke.Abstractions/Portals/IPortalAliasInfo.cs

```diff
@@ -1,27 +1,54 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information
+
 namespace DotNetNuke.Abstractions.Portals
 {
-    using System.Data;
-    using System.Xml;
-    using System.Xml.Schema;
+    using DotNetNuke.Abstractions.Urls;
 
+    /// <summary>
+    /// Portal Alias Info interface.
+    /// </summary>
     public interface IPortalAliasInfo
     {
-        //BrowserTypes BrowserType { get; set; }
-        string CultureCode { get; set; }
-        string HTTPAlias { get; set; }
+        /// <summary>
+        /// Gets or sets the http alias.
+        /// </summary>
+        string HttpAlias { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Portal Alias ID.
+        /// </summary>
+        int PortalAliasId { get; set; }
```

Comment:
@mitchelsellers this rename from `PortalAliasID` to `PortalAliasId` it technically breaking.  Is this one we can let slide b/c it's new enough, or do we need to have a deprecated `PortalAliasID` property on the interface also?

Reply:
I think we just leave it out